### ISSUE TITLE
gnt-instance console failed on el8

### ISSUE THREAD

**Issue by jfut** [jfut/ganeti-rpm#40]
```bash
# gnt-instance console vmtest1.example.org
2021/02/21 13:05:08 socat[3889] E tcsetattr(5, TCSADRAIN, 0x56011dfaf980): Inappropriate ioctl for device
Connection to centos8node01.example.org closed.
Failure: command execution error:
Connection to console of instance vmtest1.example.org failed, please check cluster configuration

# socat -ddd STDIO,raw,echo=0,escape=0x1d UNIX-CONNECT:/var/run/ganeti/kvm-hypervisor/ctrl/vmtest1.example.org.monitor
2021/02/21 13:04:42 socat[3826] I socat by Gerhard Rieger and contributors - see www.dest-unreach.org
2021/02/21 13:04:42 socat[3826] I This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit. (http://www.openssl.org/)
2021/02/21 13:04:42 socat[3826] I This product includes software written by Tim Hudson (tjh@cryptsoft.com)
2021/02/21 13:04:42 socat[3826] I setting option "raw"
2021/02/21 13:04:42 socat[3826] I setting option "echo" to 0
2021/02/21 13:04:42 socat[3826] I setting option "escape" to 29
2021/02/21 13:04:42 socat[3826] N reading from and writing to stdio
2021/02/21 13:04:42 socat[3826] N opening connection to AF=1 "/var/run/ganeti/kvm-hypervisor/ctrl/vmtest1.example.org.monitor"
2021/02/21 13:04:42 socat[3826] I socket(1, 1, 0) -> 5
2021/02/21 13:04:42 socat[3826] E tcsetattr(5, TCSADRAIN, 0x55db2ba1c980): Inappropriate ioctl for device
2021/02/21 13:04:42 socat[3826] N exit(1)
2021/02/21 13:04:42 socat[3826] I shutdown(5, 2)
```

**Comment by jfut** [comment]
https://bugs.launchpad.net/ubuntu/+source/socat/+bug/1883957

**Comment by jfut** [comment]
A release of socat-1.7.4.1-1, which includes a fix for this issue, is planned for RHEL 8.4.

https://bugzilla.redhat.com/show_bug.cgi?id=1914989

**Comment by jfut** [comment]
socat 1.7.3.4-1 RPM released!

https://github.com/jfut/ganeti-rpm/releases/tag/v3.0.1-1-socat-1.7.3.4-1

Update:

`dnf --enablerepo=epel,integ-ganeti update socat`
